dwave_networkx/drawing/chimera_layout.py

```diff
@@ -190,30 +190,30 @@ def _xy_coords(i, j, u, k):
 
 
 def draw_chimera(G, **kwargs):
-    """Draws graph G in a Chimera cross topology.
+    """Draw graph ``G`` in a Chimera cross topology.
 
-    If `linear_biases` and/or `quadratic_biases` are provided, these
-    are visualized on the plot.
+    Linear and quadratic biases are visualized on the plot as specified
+    in the ``linear_biases`` and ``quadratic_biases`` parameters.
 
     Parameters
     ----------
     G : NetworkX graph
-        Should be a Chimera graph or a subgraph of a Chimera graph.
+        :term:`Chimera` :term:`graph` or a :term:`subgraph` of a Chimera graph.
 
     linear_biases : dict (optional, default {})
-        A dict of biases associated with each node in G. Should be of
-        form {node: bias, ...}. Each bias should be numeric.
+        A dict of biases associated with each node in ``G`` and of
+        the form ``{node: bias, ...}``. Each bias is numeric.
```

Comment:
```suggestion
        Linear biases for all nodes of ``G``, as a dict of form ``{node: bias, ...}``, 
        where ``bias`` is numeric.
```

We prefer to start the description with what the parameter means ("linear biases") rather than the data type ("dict"). The above is more consistent with our docs

Also applies elsewhere below